test/color/test_colormap.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+from kornia.color import AUTUMN, ApplyColorMap, apply_colormap
+from kornia.core import tensor
+from kornia.testing import BaseTester, assert_close, tensor_to_gradcheck_var
+
+
+def test_autumn(device, dtype):
+    cm = AUTUMN(num_colors=254, device=device, dtype=dtype)
+    colors = cm.colors
+
+    actual = colors[..., 0]
+
+    expected = tensor([1, 0, 0], device=device, dtype=dtype)
+
+    assert_close(actual, expected)
+
+    actual = colors[..., 127]
+    expected = tensor([1, 0.5019997358322144, 0], device=device, dtype=dtype)
+    assert_close(actual, expected)
+
+    actual = colors[..., -1]
+    expected = tensor([1, 1, 0], device=device, dtype=dtype)
+    assert_close(actual, expected)
+
+
+class TestApplyColorMap(BaseTester):
+    def test_smoke(self, device, dtype):
+        input_tensor = tensor([[[0, 1, 3], [25, 50, 63]]], device=device, dtype=dtype)
+
+        expected_tensor = tensor(
+            [
+                [[1, 1, 1], [1, 1, 1]],
+                [[0, 0.01587301587301587, 0.04761904761904762], [0.3968253968253968, 0.7936507936507936, 1]],
+                [[0, 0, 0], [0, 0, 0]],
+            ],
+            device=device,
+            dtype=dtype,
+        )
+        cm = AUTUMN(device=device, dtype=dtype)
+        actual = apply_colormap(input_tensor, cm)
+
+        self.assert_close(actual, expected_tensor)
+
+    def test_exception(self, device, dtype):
+        cm = AUTUMN(device=device, dtype=dtype)
+        with pytest.raises(TypeError):
+            apply_colormap(torch.rand(size=(5, 1, 1), dtype=dtype, device=device), cm)
+
+    @pytest.mark.parametrize("shape", [(2, 1, 4, 4), (1, 4, 4), (4, 4)])
+    def test_cardinality(self, shape, device, dtype):
+        cm = AUTUMN(device=device, dtype=dtype)
+        input_tensor = torch.randint(0, 63, shape, device=device, dtype=dtype)
+        actual = apply_colormap(input_tensor, cm)
+
+        if len(shape) == 4:
+            expected_shape = (shape[0], 3, shape[-2], shape[-1])
+        else:
+            expected_shape = (3, shape[-2], shape[-1])
+
+        assert actual.shape == expected_shape
+
+    def test_gradcheck(self, device, dtype):
```

Comment:
I don't know what I can do to solve the jacobian mismatch for gradcheck -- I appreciate if someone have an idea about it

Reply:
i disable the [differentiability test](https://github.com/kornia/kornia/pull/1996/commits/99b4cb3c47cd306055365915dbdab8d510914e5a)
[99b4cb3](https://github.com/kornia/kornia/pull/1996/commits/99b4cb3c47cd306055365915dbdab8d510914e5a)